### PR TITLE
Chris

### PR DESCRIPTION
**Pull request by Cpolackal**
In the class DressedQuantumNet(nn.Module), I added:
self.bn = nn.BatchNorm1d(n_qubits)
This was in the constructor.

I also added: 
q_out = self.bn(q_out)
Right before feeding into the model

### CODE REVIEW

**Comment by Cpolackal** [comment]
Name: Chris Polackal
Email: polackal@terpmail.umd.edu
